packages/rest/lib/RestManager.ts

```diff
@@ -53,60 +66,107 @@ export class RestManager {
         return this.proxyURL ?? `https://www.guilded.gg/api/v${this.version}`;
     }
 
+    /**
+     * Generate obfuscated token. It replaces every char in a non-even index with X. I'm not very creative.
+     */
+    get obfuscatedToken(): string {
+        return this.token.split("").reduce((prev, curr, index) => index % 2 ? prev + curr : prev + "X", "")
```

Comment:
i would just obfuscate all of the characters here. it may not make much of a practical difference if only half of the token is shown as far as security is concerned; however, we take this kind of security seriously, so i'm just being overly paranoid here